packages/dashboard/src/app/serializers/test/templates.js

```diff
@@ -32,27 +32,14 @@ describe('reshapeTemplateObject', () => {
       pages: [{ id: 1 }, { id: 2 }, { id: 3 }, { id: 4 }],
       creationDate: '2020-03-26T20:57:24',
       modified: '2020-03-26T20:57:24',
+      status: 'template',
     };
 
-    const reshapedObj = reshapeTemplateObject(responseObj, 'example.com/');
+    const reshapedObj = reshapeTemplateObject(responseObj, false);
 
     expect(reshapedObj).toMatchObject({
       id: 1,
       slug: 'template-1-slug',
-      postersByPage: {
-        0: {
-          webp: 'example.com/images/templates/template-1-slug/posters/1.webp',
-          png: 'example.com/images/templates/template-1-slug/posters/1.png',
-        },
-        1: {
-          webp: 'example.com/images/templates/template-1-slug/posters/2.webp',
-          png: 'example.com/images/templates/template-1-slug/posters/2.png',
-        },
-        2: {
-          webp: 'example.com/images/templates/template-1-slug/posters/3.webp',
-          png: 'example.com/images/templates/template-1-slug/posters/3.png',
-        },
-      },
```

Comment:
Since this no longer happens in this serializer, I removed the tests here. Should I create new test for `getTemplates`? @BrittanyIRL

Reply:
There currently aren't any tests in templates for `getTemplates`.

Reply:
🤔  we've only got tests for the `raw` templates right now, i think adding in a test to make sure they are formatting accurately is a great idea.